drools-wb-screens/drools-wb-guided-rule-editor/drools-wb-guided-rule-editor-client/src/main/java/org/drools/workbench/screens/guided/rule/client/editor/ConstraintValueChangeHandler.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.drools.workbench.screens.guided.rule.client.editor;
+
+import java.util.Arrays;
+
+import com.google.gwt.event.logical.shared.ValueChangeEvent;
+import com.google.gwt.user.client.Command;
+import org.drools.workbench.models.datamodel.rule.BaseSingleFieldConstraint;
+
+public class ConstraintValueChangeHandler
+        implements com.google.gwt.event.logical.shared.ValueChangeHandler<String> {
+
+    private final BaseSingleFieldConstraint constraint;
+    private final Command onChangeCommand;
+    private final String in = "in";
+    private final String not_in = "not in";
+
+    public ConstraintValueChangeHandler(final BaseSingleFieldConstraint constraint,
+                                        final Command onChangeCommand) {
+        this.constraint = constraint;
+        this.onChangeCommand = onChangeCommand;
+    }
+
+    @Override
+    public void onValueChange(ValueChangeEvent<String> event) {
+
+        if (Arrays.asList(in, not_in).contains(constraint.getOperator())) {
```

Comment:
Please use `Arrays.asList(OperatorsOracle.EXPLICIT_LIST_OPERATORS)` instead of `Arrays.asList(in, not_in)`

Reply:
Thanks @jomarko that was something I was meaning to check!